src/SimpleDatabase.php

```diff
@@ -288,6 +288,14 @@ public function getPdoBasedOnQueryType(string $sql): PDO
         }
     }
 
+    /**
+     * @return bool
+     */
+    public function inTransaction(): bool
```

Comment:
Please move up in file to be together with other transaction-related methods please.